src/components/EventUserTable.js

```diff
@@ -81,70 +180,99 @@ export class EventUserTable extends Component {
-        title={`${this.props.event.ename} Attendance`}
-        columns={[
-          { title: "First Name", field: "fname" },
-          { title: "Last Name", field: "lname" },
-          {
-            title: "Student Number",
-            field: "id",
-            type: "numeric",
-            sorting: false
-          },
-          { title: "Email", field: "email", sorting: false },
-          {
-            title: "Registration Status",
-            field: "registrationStatus",
-            sorting: false,
-            render: rowData => (
-              <div>
-                <Select
-                  value={rowData.registrationStatus}
-                  onClick={event => changeRegistration(event, rowData)}
-                  style={{
-                    backgroundColor:
-                      rowData.registrationStatus === REGISTRATION_STATUS.CHECKED_IN
-                        ? "#54D26E"
-                        : rowData.registrationStatus === REGISTRATION_STATUS.WAITLISTED
-                          ? "#F7D055"
-                          : rowData.registrationStatus === REGISTRATION_STATUS.CANCELLED
-                            ? "#E15453"
-                            : "#FFF",
-                    paddingLeft: "10px"
-                  }}>
-                  <MenuItem value={REGISTRATION_STATUS.WAITLISTED}>Waitlisted</MenuItem>
-                  <MenuItem value={REGISTRATION_STATUS.CHECKED_IN}>Checked in</MenuItem>
-                  <MenuItem value={REGISTRATION_STATUS.REGISTERED}>Registered</MenuItem>
-                  <MenuItem value={REGISTRATION_STATUS.CANCELLED}>Cancelled</MenuItem>
-                </Select>
-              </div>
-            )
-          }
-        ]}
-        data={this.state.rows}
-        // Configure options for the table
-        options={{
-          search: true,
-          draggable: false,
-          padding: "dense",
-          pageSize: 15,
-          pageSizeOptions: [15, 50, 100],
-          actionsColumnIndex: 5,
-          exportButton: true,
-          headerStyle: {
-            fontWeight: "bold"
-          },
-          rowStyle: rowData => ({
+      <div>
+        <div style={styles.stats}>
+          <Typography style={styles.stat}>Registered: {this.state.registered}</Typography>
+          <Typography style={styles.stat}>Checked in: {this.state.checkedIn}</Typography>
+          <Typography style={styles.stat}>Waitlisted: {this.state.waitlisted}</Typography>
+          <Typography style={styles.stat}>Cancelled: {this.state.cancelled}</Typography>
+          <Typography style={styles.stat}>Total: {this.state.registered + this.state.checkedIn + this.state.waitlisted + this.state.cancelled}</Typography>
+        </div>
+        <div style={styles.stats}>
+          <Typography style={styles.stat}>Faculty: </Typography>
+          {Object.keys(this.state.faculties).map(key => (<Typography key={key} style={styles.stat}>{key}: {this.state.faculties[key]}</Typography>))}
```

Comment:
This is fine, fyi its possible to do Object.entries and your map will have to params `.map((key, value) => {...})`